bin/plot_krun_results

```diff
@@ -138,7 +138,7 @@ def add_instr_data(data_dictionary, key, machine, instr_dir):
 def main(is_interactive, data_dcts, plot_titles, window_size, outfile,
          xlimits, with_outliers, unique_outliers, changepoints, changepoint_means,
          median=False, tukey=False, inset_xlimit=None, one_page=False,
-         legend_off=True, cycles_ylimits=None):
+         legend_off=True, core_cycles=(0,1,2,3), cycles_ylimits=None):
```

Comment:
This assumes we have 5 cores. Is that what we want? Does it do the right thing on openbsd, with no core data?